### PR TITLE
DF 1443 : submit created PoWEs if the ControlPeriod is in the past

### PR DESCRIPTION
**Pull request by emina-str**
UI : https://github.com/alv-ch/alv-portal/pull/595

### CODE REVIEW

File: application/src/main/java/ch/admin/seco/alv/service/onlineform/application/npa/ProofOfWorkEffortsApplicationService.java [hunk 1]
```diff
@@ -126,6 +126,9 @@ public ProofOfWorkEffortsDto create(CreateProofOfWorkEffortsDto createProofOfWor
 		}
 		ProofOfWorkEfforts proofOfWorkEfforts = this.proofOfWorkEffortsRepository.save(toProofOfWorkEfforts(createProofOfWorkEffortsDto));
 		proofOfWorkEfforts.onCreated(this.dateCalculator);
+		if (proofOfWorkEfforts.getControlPeriod().isInThePast()) {
```
Comment (pavax) [inline application/src/main/java/ch/admin/seco/alv/service/onlineform/application/npa/ProofOfWorkEffortsApplicationService.java]: We should not rely on the isInThePast() method. For exmaple: if a user registers at 01.10.2019 the NPA 2019-09 must still be open as it get's submittet at the 05.10.2019.

Comment (emina-str) [reply application/src/main/java/ch/admin/seco/alv/service/onlineform/application/npa/ProofOfWorkEffortsApplicationService.java]: Correct. Implemented by checking nextSubmissionDate to currentDate.

File: domain/src/test/java/ch/admin/seco/alv/service/onlineform/domain/npa/ProofOfWorkEffortsRepositoryTest.java [hunk 2]
```diff
@@ -132,6 +130,8 @@ public void testFindPrevious() {
 	@Test
 	public void testFindByWorkEffortId() {
 		// given
+		TimeMachine.useFixedClockAt(LocalDate.parse("2019-01-01").atStartOfDay());
```
Comment (pavax) [inline domain/src/test/java/ch/admin/seco/alv/service/onlineform/domain/npa/ProofOfWorkEffortsRepositoryTest.java]: I guess that line is not needed?

File: infrastructure/src/main/java/ch/admin/seco/alv/service/onlineform/infrastructure/scheduler/SubmitProofOfWorkEffortsTask.java [hunk 3]
```diff
@@ -30,8 +28,6 @@
 	}
 
 	@Scheduled(cron = "${onlineService.submitProofOfWorkEfforts.cron}")
-	@LeaderAware(TASK_NAME)
```
Comment (pavax) [inline infrastructure/src/main/java/ch/admin/seco/alv/service/onlineform/infrastructure/scheduler/SubmitProofOfWorkEffortsTask.java]: revert

**Pull request merged**